packages/core/src/common/theming-protocol.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+export interface Theme {
```

Comment:
I don't see a reason for a protocol (frontend <-> backend) communication. Also those objects need to be JSON (i.e. no functions).

Reply:
I was expecting themes to be loadable from disk later, like some kind of `~/.theia/themes` folder where the user could drop whatever he wanted.

Now indeed this `Theme` interface is in the wrong file.